setup.py

```diff
@@ -50,15 +50,14 @@
     url='https://github.com/ethereum/eth-tester',
     include_package_data=True,
     install_requires=[
-        "toolz>0.8.2,<1;implementation_name=='pypy'",
-        "cytoolz>=0.8.2,<1.0.0;implementation_name=='cpython'",
+        "eth-abi>=1.0.0,<2",
```

Comment:
@carver do we want to bump the minimum to 1.2.1? Also, we probably need to rerun the last build to pick up the new release.

Reply:
Yeah, I think that's fine :+1: -- I tried rerunning, but it still didn't pick up 1.2.1 (until I ssh in, and then `~/.local/bin/tox -elint` runs fine.

Anyway, changing the eth-abi req to`>=1.2.1,<2` will trigger it again cleanly.

Reply:
Fixed: https://github.com/ethereum/eth-tester/pull/138/commits/23bd953568682ab4ef7bc833c3c4301a6abdadb9